server/swap/swap.go

```diff
@@ -1497,6 +1493,11 @@ func (s *Swapper) processRedeem(msg *msgjson.Message, params *msgjson.Redeem, st
 		// Neither party's fault. Continue.
 	}
 
+	// Credit the user for completing the swap, adjusting the user's score.
+	if actor.user != counterParty.user || newStatus == order.MatchComplete { // if user is both sides, only credit on MatchComplete (taker done too)
+		s.authMgr.SwapSuccess(actor.user, db.MatchID(match.Match), match.Quantity, redeemTime) // maybe call this in swapDone callback
+	}
```

Comment:
Fairly inconsequential for a success, but moved below the DB store for consistency.